index.js

```diff
@@ -302,14 +306,16 @@ function createPointCluster(p, id) {
         x: lngX(coords[0]), // projected point coordinates
         y: latY(coords[1]),
         zoom: Infinity, // the last zoom the point was processed at
-        id: id, // index of the source feature in the original input array
+        index: id, // index of the source feature in the original input array,
+        id: p.id, // id of the source feature
```

Comment:
We don't have to store original feature ids in the cluster objects because we can trace them back from original features (i.e. `points[p.index].id`).